app/controllers/planning_applications/neighbour_letters_controller.rb

```diff
@@ -55,6 +55,9 @@ def send_letters
       end
     rescue Neighbour::AddressValidationError => e
       redirect_address_validation_error(e)
+    rescue ActiveRecord::RecordInvalid => e
+      flash[:alert] = e
+      redirect_to planning_application_consultation_neighbour_letters_path(@planning_application)
```

Comment:
This is the same behaviour as the `redirect_address_validation_error` method - I don't think it's a good idea to set the exception in the flash as it'll just use a `to_s` on the instance which isn't very user friendly. Would be better to do something like this:

``` ruby
rescue Neighbour::AddressValidationError
  redirect_after_rescue(t(".errors.address_validation"))
rescue ActiveRecord::RecordInvalid
  redirect_after_rescue(t(".errors.record_invalid"))
end

def redirect_after_rescue(alert)
  redirect_to planning_application_consultation_neighbour_letters_path(@planning_application), alert: alert
end
```